NEW Link type icon (#170)
silverstripe/silverstripe-linkfield: README.md

Link::remove_extension(Versioned::class);
```

## Additional features

> The developer can customize the position of the link type in the menu by setting the `$menu_priority` value. The priority is set ascending, i.e. the link will be displayed in the menu above if the priority value is less. By default, the highest priority has SiteTreeLink equal to zero.

Reviewer: ```suggestion
The developer can customise the position of the link type in the menu by setting the `$menu_priority` value. The priority is in ascending order (i.e. a link with a higher priority value will be displayed lower in the list).
```
"customise" instead of "customize" - the z is american english.
I've reworded this to use much simpler language, and I've removed the bit about `SiteTreeLink` equaling zero because people can just look at the config in the code to see that.

Author: Updated

> The developer can also set an icon that will correspond to a specific type of link by setting the value of the `$icons` property. This name must correspond to the name of the class of the icon that needs to be used.

Reviewer: ```suggestion
The developer can also set an icon that will correspond to a specific type of link by setting the value of the `$icon` configuration property. The value of this configuration corresponds to the css class of the icon to be used.
```

Author: Updated
